mdc-101/complete/index.html

```diff
@@ -25,16 +25,16 @@ <h1>SHRINE</h1>
   </section>
 
   <form action="home.html">
-    <div class="mdc-text-field username">
-      <input type="text" class="mdc-text-field__input" id="username-input" name="username" required>
-      <label class="mdc-floating-label" for="username-input">Username</label>
+    <label class="mdc-text-field username">
```

Comment:
Can we merge this to `develop` branch? It seems like this change is based on unreleased MDC version.

Reply:
I don't think we have a develop branch in this repo, want me to make one?

Reply:
We can create a new branch. I think `master` branch is directly used in our catalog. Introducing these changes would break the codelabs for users who are still on v4.0.